properties/P000133.md

```diff
@@ -2,7 +2,7 @@
 uid: P000133
 name: LOTS
 aliases:
-  - Linearly orderable
+  - Orderable
```

Comment:
```suggestion
  - Linearly orderable
```

Changing the alias to just "orderable" would make P133 not appear in a search for "linearly", so I think it's best to keep the keyword in the alias. (Perhaps we should eventually support a separate list of keywords that don't appear in the title or aliases but we want to be triggered by searches.)